.github/workflows/default.yml

```diff
@@ -445,6 +445,13 @@ jobs:
       with:
         name: local-proxy
         path: dist
+    - name: SBOM report generation
+      uses: ./.github/workflows/sbom.yml
```

Comment:
Could we parallelize this by separating it out as a separate job?

Reply:
I think that parallelizing simple tasks will take more time for runner init, so I tend not to separate them for non-complex tasks.
In the case of reusable workflow, runner setup is needed anyway, so it would be better to separate it!